opentelekomcloud/services/dms/resource_opentelekomcloud_dms_group_v1.go

```diff
@@ -22,6 +22,8 @@ func ResourceDmsGroupsV1() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 
+		DeprecationMessage: "please use `opentelekomcloud_dms_instance_v1` resource instead",
```

Comment:
add `support will be discontinued in favor of DMS Kafka Premium` here. the explanation missing

Reply:
explanation added, please check